src/Model/Scope/CompoundCondition.php

```diff
@@ -178,14 +178,14 @@ public function toWords(Model $model = null): string
      */
     public static function createAnd(...$conditions)
     {
-        return new self($conditions, self::AND);
+        return new static($conditions, self::AND);
```

Comment:
`self` is intentional as using `Model\Scope::createAnd` brings unexpected results

Reply:
I will fix